openedx/core/djangoapps/theming/helpers.py

```diff
@@ -52,3 +71,142 @@ def get_themed_template_path(relative_path, default_path, **kwargs):
     if is_stanford_theming_enabled and not is_microsite:
         return relative_path
     return microsite.get_template_path(default_path, **kwargs)
+
+
+def get_template_path_with_theme(relative_path):
```

Comment:
We really need to have tests for each of these methods in order to cover all of the various logical branches -- perhaps @saleem-latif can assist with this work, which would help him to build up more context around the implementation.